infra/routes/word_similarity_cache_generator/lambdas/index.py

```
@@ -620,8 +620,12 @@ def process_receipt(work_item):
                 price = line_total or unit_price
                 size = infer_size(product_text, price)
 
-                # Convert lines to dict format
-                lines_dict = [line_to_dict(line) for line in details.lines]
+                # NOTE: details.lines (the full per-receipt line list, ~75 lines
+                # × 69 receipts ≈ 2.8 MB across the response) is intentionally
```

Ruff (0.15.13)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)